manifests/base/applicationset-controller/argocd-applicationset-controller-deployment.yaml

```diff
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   labels:
     app.kubernetes.io/name: argocd-applicationset-controller
-    app.kubernetes.io/part-of: argocd-applicationset
+    app.kubernetes.io/part-of: argocd
     app.kubernetes.io/component: controller
```

Comment:
I'm not familiar with these labels... is `controller` going to clash with our other components? In other words, should we rename it `applicationset-controller`?

Reply:
Good point! Given `app.kubernetes.io/component: application-controller` is used in the application-controller, I agree we should rename it to `applicationset-controller`